### ISSUE TITLE
vgg_cifar10_train.ipynb training failed.

### ISSUE THREAD

**Issue by NienfengYao** [NienfengYao/TensorFlow_Study#1]
The [vgg16f_training_cifar10.ipynb](https://github.com/NienfengYao/TensorFlow_Study/blob/master/vgg16/vgg16_final/vgg16f_training_cifar10.ipynb) training is failed. I can't get good accuracy.

**Comment by NienfengYao** [comment]
I add batch normalization in each hidden layer. Now we can get good accuracy.
But in fact, why we can't get good accuracy in the original model (w/o batch normalization).
